.eslintrc

```diff
@@ -102,6 +101,30 @@
       "env": {
         "jest": true
       }
+    },
+    {
+      "files": ["src/**", "pages/**", "test/**", "scripts/**"],
+      "rules": {
+        "simple-import-sort/imports": [
+          "warn",
+          {
+            "groups": [
+              // External packages come first.
+              ["^react", "^\\w", "^(?!@cloudscape)@?\\w"],
```

Comment:
Wouldn't the last item in this array be enough?

```
["^(?!@cloudscape)@?\\w"]
```

This regex alone already seems to do what we need: it matches any letter-starting string, optionally prefixed with `@`, including `react`, but excluding those starting with `@cloudscape`

Reply:
addressed